### PR TITLE
Add "Keeping your instance synced" section to "Upgrade Guide" page

### PR DESCRIPTION
**Pull request by sussy-code**
[Discord Context](https://discord.com/channels/871713465100816424/871713465578979351/1220747802225934417), Adds a guide with 2 ways to keep an instance up to date.

### CODE REVIEW

**Review by zisra — APPROVED** [review]

File: content/3.client/5.upgrade.md [hunk 1]
```diff
@@ -1,8 +1,66 @@
 ---
-title: 'Upgrade guide'
+title: 'Update guide'
 ---
 
-# Upgrade guide
+# Keeping your instance synced
+
+Movie-web's github is constantly being updated. If you want to keep your instance synced to the most recent version, you will need to follow one of the two guides below.
```
Comment (JorrinKievit) [inline content/3.client/5.upgrade.md:7]:
```suggestion
Keeping your instance up-to-date with the latest features and bug fixes can enhance your instance's functionality and ensure it stays current. When updates are released, you have the option to adopt them using either one of the guides. Below, you willl find a manual and an automatic guide for updating your instance.
```

File: content/3.client/5.upgrade.md [hunk 2]
```diff
@@ -1,8 +1,66 @@
...
+
+## Manual update
+
+You can manually update by typing the below lines into a terminal at the root of your instance
```
Comment (JorrinKievit) [inline content/3.client/5.upgrade.md:11]:
```suggestion
You can manually update by executing the following commands at the root of your instance:
```

File: content/3.client/5.upgrade.md [hunk 3]
```diff
@@ -1,8 +1,66 @@
...
+    - uses: actions/checkout@v4
+      with:
+        persist-credentials: false
+    - name: repo-sync
+      uses: repo-sync/github-sync@v2
+      with:
+        source_repo: "" # Change this to the repo you want to sync from
+        source_branch: "" # Change this to the branch you want to sync from
+        destination_branch: "" # Change this to the branch you want to sync to
+        github_token: ${{ secrets.PAT }}
```
Comment (JorrinKievit) [inline content/3.client/5.upgrade.md:52]: Can't the automatic secret GITHUB_TOKEN be used here? https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Comment (sussy-code) [reply content/3.client/5.upgrade.md]: Changed yaml file to the below snippet
```yaml
# File: .github/workflows/sync.yml
name: Sync fork

on:
  schedule:
  - cron: "*/5 * * * *"
  push:
    branches:
    - "*"
    paths:
    - .github/workflows/sync.yml

jobs:
  sync:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v3
    - run: gh repo sync reponame/forkname
      env:
        GH_TOKEN: ${{ github.token }}
    - uses: gautamkrishnar/keepalive-workflow@v1
```

**Comment by sussy-code** [comment]
pls merge

**Review by binaryoverload — CHANGES_REQUESTED** [review]
Thanks for this

As the primary recommendation, I would like people to fork the repo instead as it's a much less error prone method of keeping up to date. Would you be willing to update this page and the client setup page to include that?

File: content/3.client/5.upgrade.md [hunk 4]
```diff
@@ -1,8 +1,65 @@
...
 ---
 
-# Upgrade guide
+# Keeping your instance synced
+
+Keeping your instance up-to-date with the latest features and bug fixes can enhance your instance's functionality and ensure it stays current. When updates are released, you have the option to adopt them using either one of the guides. Below, you willl find a manual and an automatic guide for updating your instance.
+
+## Manual update
+
+You can manually update by executing the following commands at the root of your instance:
```
[the remark this answers is not in the document]
Comment (sussy-code) [reply content/3.client/5.upgrade.md:11]: You can manually update by executing the following commands in the root directory of the repository you have created?

File: content/3.client/5.upgrade.md [hunk 5]
```diff
@@ -1,8 +1,65 @@
...
+
+```bash
+git remote add movie-web https://github.com/movie-web/movie-web.git
+git fetch movie-web
+# Change `dev` to `master` if you want a stable experience
+git merge movie-web/dev --allow-unrelated-histories
+git push -f # Force push to your origin main branch
```
[the remark this answers is not in the document]
Comment (sussy-code) [reply content/3.client/5.upgrade.md:18]: The git push -f command is necessary due to the --allow-unrelated-histories option in the git merge command.

File: content/3.client/5.upgrade.md [hunk 6]
```diff
@@ -33,11 +33,14 @@ jobs:
...
     steps:
     - uses: actions/checkout@v3
-    - run: gh repo sync <OWNER>/<FORK>
+    - run: gh repo sync <OWNER>/<FORK> # Replace the placeholders within the < >
       env:
         GH_TOKEN: ${{ github.token }}
     - uses: gautamkrishnar/keepalive-workflow@v1
 ```
+::alert{type="info"}
+Github action authored by [Toon](https://github.com/Toon-arch)
```
Comment (binaryoverload) [inline content/3.client/5.upgrade.md:42]: Remove this please, it's irrelevant to the docs

**Review by binaryoverload — CHANGES_REQUESTED** [review]

File: content/3.client/5.upgrade.md [hunk 7]
```diff
@@ -33,11 +33,14 @@ jobs:
...
+::
```
Comment (binaryoverload) [inline content/3.client/5.upgrade.md:43]:
```suggestion
```

**Review by binaryoverload — APPROVED** [review]

**Pull request merged**